bcda/cli.go

```diff
@@ -82,6 +83,9 @@ func importCCLF8(filePath string) error {
 }
 
 func importCCLF9(filePath string) error {
```

Comment:
Wondering if this will work for us: 
https://github.com/CMSgov/bcda-app/blob/54a1de48cb7263e31d5b1e570019a3a5fdcea2e6/bcda/main.go#L304-L331

Basically, this requires that the caller (for sake of conversation, lets say its Jenkins on a cron job) knows the name of the file to be ingested.  I'm not sure that is going to be feasible, mainly because we don't want Jenkins (or any caller really) to have access to the directory where these files will live - so how will they know the file name?  
I wonder if we need to consider a single CLI command called something like `importCCLF` which accepts a directory path as an argument.  Internally, that function will iterate through all the files in the dir (could be 0 files), determine if each file is cclf0/8/9, and then call the internal functions we have written to process 0, 8, and 9.  Basically, processing all files it finds instead of just a single file at a time.

Curious to hear thoughts from @embh and @knollfear on this.

Reply:
I like the idea of passing a directory path to a general CCLF import command. The imports of all three files are related, with CCLF0 for validating the rows in the others, and the application could clean up all the files together (or at least indicate that they're ready for cleanup, if we decide to make that a separate job). Making a new command to check a directory and determine which CCLF # each file is sounds like it might be worthy of a new ticket.

Reply:
good idea. If it's implemented in this ticket or a new one, doesn't matter to me :)

Reply:
I created this ticket https://jira.cms.gov/browse/BCDA-1184